donut/modules/directory_search/templates/search_results.html

```diff
@@ -1,10 +1,8 @@
 {% extends 'layout.html' %}
 {% block page %}
 	<div class='container theme-showcase jumbotron' style='float: none; padding-bottom: 25px'>
-		<button class='btn' onclick='history.back()'>
-			&lt; Back
-		</button>
-		<div>Your search returned {{ users|length or 'no' }} results</div>
+		<a class='btn btn-info' href='{{ url_for("directory_search.directory_search") }}'>Back</a>
```

Comment:
`history.back()` was intentional so the browser will restore all the form inputs' values

Reply:
history.back() goes back to the last results page instead of the search form